src/pudl/workspace/datastore.py

```diff
@@ -60,8 +60,12 @@ def __init__(self, datapackage_json: dict, dataset: str, doi: ZenodoDoi):
     def get_resource_path(self, name: str) -> str:
         """Returns zenodo url that holds contents of given named resource."""
         res = self._get_resource_metadata(name)
-        # remote_url is sometimes set on the local cached version of datapackage.json
-        # so we should be using that if it exists.
+        # In older cached archives, "remote_url" was used to refer to the original path
+        # to the file, while the canonical "path" field was updated by the datastore
+        # to refer to the local path to the associated file relative to the location of
+        # datapackage.json. This behavior is deprecated and no longer used, but we need
+        # to retain this logic to support older cached archives, e.g. censusdp1tract
+        # which hasn't changed since 2020.
```

Comment:
A clarification that goes along with the changes in https://github.com/catalyst-cooperative/pudl-archiver/pull/303